webview-ui/src/components/ProfileManager/options.ts

```diff
@@ -0,0 +1,144 @@
+export const targetOptions = [
```

Comment:
Need to double check that we still have the ticket to pull these from the configured rules. That may impact the order in the form.

1. Name + description
2. CustomRules
3. UseDefaultRulesets
4. Configure sources+targets (we will need some way to load the rules and pull the source/target labels out).

Reply:
https://github.com/konveyor/editor-extensions/issues/305